### DISCUSSION TITLE
Facing problem in contributing to framer motion open source

### DISCUSSION THREAD

**Discussion by Arun-chaitanya** [motiondivision/motion#2214] [Q&A]
Hi There,
I want to contribute to Framer's open source. 
1) I cloned the "**motion**" repository. And I ran the `yarn install` command on the root repo. 
2) Now next step mentioned is "Run `make bootstrap` in the repo root." But this is giving me an error - `make: *** No rule to make target `bootstrap'. Stop.`. 
3) If I do `npm run dev`, I am getting till `created dist/framer-motion.js in 965ms`.
<img width="806" alt="Screenshot 2023-07-02 at 3 36 51 PM" src="https://github.com/framer/motion/assets/44457256/6f1e296d-820a-498e-9655-0f9bb45ad80a">

**Question)** Can you tell me how to start the repo in my local?

Thanks.

**Comment by lochie** [comment]
@Arun-chaitanya I know this was posted a long time ago, but I also had similar issue and was resolved by updating node.
